R/g.analyse.perday.R

```diff
@@ -521,29 +521,23 @@ g.analyse.perday = function(ndays, firstmidnighti, time, nfeatures,
                       rr1 = matrix(0, length(varnum), 1)
                       p = which(varnum * UnitReScale >= params_phyact[["mvpathreshold"]][mvpai]); rr1[p] = 1
                       getboutout = g.getbout(x = rr1, boutduration = boutduration,
-                                             boutcriter = params_phyact[["boutcriter"]],
-                                             closedbout = params_phyact[["closedbout"]],
-                                             bout.metric = params_phyact[["bout.metric"]], ws3 = ws3)
-                      mvpa[4] = length(which(getboutout$x == 1)) / (60/ws3) #time spent MVPA in minutes
+                                             boutcriter = params_phyact[["boutcriter"]], ws3 = ws3)
+                      mvpa[4] = length(which(getboutout == 1)) / (60/ws3) #time spent MVPA in minutes
                       
                       # METHOD 5: time spent above threshold 5 minutes
                       boutduration = params_phyact[["mvpadur"]][2] * (60/ws3) #per five minutes
                       rr1 = matrix(0, length(varnum), 1)
                       p = which(varnum * UnitReScale >= params_phyact[["mvpathreshold"]][mvpai]); rr1[p] = 1
                       getboutout = g.getbout(x = rr1, boutduration = boutduration,
-                                             boutcriter = params_phyact[["boutcriter"]],
-                                             closedbout = params_phyact[["closedbout"]],
-                                             bout.metric = params_phyact[["bout.metric"]], ws3 = ws3)
-                      mvpa[5] = length(which(getboutout$x == 1))   / (60/ws3) #time spent MVPA in minutes
+                                             boutcriter = params_phyact[["boutcriter"]], ws3 = ws3)
+                      mvpa[5] = length(which(getboutout == 1))   / (60/ws3) #time spent MVPA in minutes
                       # METHOD 6: time spent above threshold 10 minutes
                       boutduration = params_phyact[["mvpadur"]][3] * (60/ws3) # per ten minutes
```

Comment:
Just realized that GGIR expects that `mvpadur` is of length = 3. Should we say this in the documentation? Or do a for look through the `mvpadur` defined by the user to make it more flexible as in part 5?

Reply:
You do not have to do this for me, but if you want to work on it then feel free to do so. Ideally, as a separate issue and PR as it does not relate to bout detection directly.